atd-vze/src/views/Crashes/RelatedRecordsTable.js

```diff
@@ -231,7 +232,6 @@ const RelatedRecordsTable = ({
                                       handleInputChange(e, updateFieldKey)
                                     }
                                   >
-                                    <option value={""}>NO DATA</option>
```

Comment:
this was just for the suspected unhoused field, but now that this field is not nullable and defaults to false we dont need this `no data` option